index.ts

```diff
@@ -214,7 +214,23 @@ namespace tss {
         private formatDiagnostics(diagnostics: ts.Diagnostic[]): string {
             return diagnostics.map((d) => {
                 if (d.file) {
-                    return 'L' + d.file.getLineAndCharacterOfPosition(d.start).line + ': ' + d.messageText;
+                    const line       = d.file.getLineAndCharacterOfPosition(d.start).line;
+                    const linePrefix = `Line ${line}: `;
+
+                    if (typeof d.messageText === 'object') {
+                        if (d.messageText.next != null) {
+                            let indentStr = '';
+                            for (let i = 0; i < linePrefix.length; i++) {
+                                indentStr += ' ';
+                            }
+
+                            d.messageText.messageText += `\n${indentStr}${d.messageText.next.messageText}`;
+                        }
+
+                        d.messageText = d.messageText.messageText;
+                    }
+
+                    return linePrefix + d.messageText;
```

Comment:
Indeed, it might be more appropriate to use this method.
Put perhaps it could be a good idea to keep some of the formating I introduced into this PR for readability ?

What do you think ?

Cheers